llama_stack/providers/adapters/inference/bedrock/bedrock.py

```diff
@@ -0,0 +1,359 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from typing import Tuple, Dict, List, Optional, Union, AsyncGenerator
+
+import boto3
+from botocore.client import BaseClient
+from llama_models.datatypes import SamplingParams
+from llama_models.llama3.api import StopReason, Message, ToolCall, CompletionMessage, ToolParamDefinition, \
+    ToolDefinition, ToolChoice, InterleavedTextMedia, ToolPromptFormat, BuiltinTool
+
+from llama_models.llama3.api.chat_format import ChatFormat
+from llama_models.llama3.api.tokenizer import Tokenizer
+from llama_models.sku_list import resolve_model
+
+from llama_stack.apis.inference import Inference, ChatCompletionResponse, \

```

Comment:
we'd prefer this to be `import *  # noqa: F403` because the APIs are meant to have only type declarations (and no code) and you should be able to just import all of them without any other consequences.

Reply:
Will update.